frontend/app/store/global.ts

```
@@ -343,7 +350,10 @@ function getOverrideConfigAtom<T extends keyof SettingsType>(blockId: string, ke
     return overrideAtom;
 }
 
-function useOverrideConfigAtom<T extends keyof SettingsType>(blockId: string, key: T): SettingsType[T] {
+function useOverrideConfigAtom<T extends keyof SettingsType>(blockId: string | null, key: T): SettingsType[T] {
+    if (blockId == null) {
+        return useAtomValue(getSettingsKeyAtom(key));
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)